plaidml/bridge/openvino/src/plaidml_plugin/ops/pad.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("pad", [](const Context& ctx) {
+  auto* layer = dynamic_cast<ngraph::opset1::Pad*>(ctx.layer);
+  IE_ASSERT((ctx.operands.size() == 3) || (ctx.operands.size() == 4));
+
+  auto I = ctx.operands.at(0);
+  std::vector<int> lo_pads;
+  auto lopadset = get_axis_vector_from_constant_operand(1, ctx.layer);
```

Comment:
This probably works, but it's a little awkward since these are padding amounts, not axes. Might be better to use an approach like StridedSlice's `cast_constant_operand`